test/framework/namespace.go

```diff
@@ -124,6 +123,18 @@ func (f *Framework) CreateNamespaceWithIPS(name string) (*corev1.Namespace, erro
 	return namespace, nil
 }
 
+// Get the first found ImagePullSecret
+func (f *Framework) getIBMCloudPullSeret() (*v1.Secret, error) {
+	var err error
+	for _, secretName := range IBMCloudSecretNames {
+		imagePullSecret, err := f.KubeClient.CoreV1().Secrets("default").Get(context.TODO(), secretName, metav1.GetOptions{})
```

Comment:
I believe this will initialise a new `err` with loop scope rather than updating the var from line 128. Thus the error on 135 will still be nil.

```suggestion
		var imagePullSecret *v1.Secret
		imagePullSecret, err = f.KubeClient.CoreV1().Secrets("default").Get(context.TODO(), secretName, metav1.GetOptions{})
```